drift/info_parser.py

```diff
@@ -0,0 +1,43 @@
+DRIFT_FACTS = {'bios_uuid', 'display_name'}
+DRIFT_HOST_ID = {'id'}
+
+
+def select_applicable_info(info_keys, hosts):
+    """
+    this pivots a list of hosts with dicts of info + values into a list of info
+    with dicts of hosts + values
+    """
+    # create dicts of host + info
+    ids_and_info = [{key: host[key] for key in (info_keys | DRIFT_HOST_ID & host.keys())}
+                    for host in hosts]
+
+    # TODO: we are assuming all info keys exist for each host. This is not a good assumption
+    available_info_names = ids_and_info[0].keys()
+
+    info_comparisons = [create_comparison(ids_and_info, info_name)
+                        for info_name in available_info_names
+                        if info_name in info_keys]
+    return info_comparisons
+
+
+def create_comparison(hosts_infos, info_name):
+    info_comparison = 'DIFFERENT'
+
+    host_values = [{x['id']: x[info_name]} for x in hosts_infos]
+
+    # map the info values down to a set. This lets us check cardinality for sameness.
+    if len({x[info_name] for x in hosts_infos}) == 1:
+        info_comparison = 'SAME'
+
+    return {'name': info_name, 'status': info_comparison, 'hosts': host_values}
+
+
+def host_metadata(host):
+    return {'id': host['id'], 'fqdn': host['fqdn'], 'last_updated': host['updated']}
+
+
+def build_comparisons(hosts_from_inventory_svc):
```

Comment:
What is svc?

Reply:
service. I guess i can flip it to be "inventory_service_hosts" or something

Reply:
Ahh, I suck at mentally interpreting acronyms when I read code.

Reply:
updtd 4 fwr acrnms